src/org/javarosa/core/io/Std.java

```diff
@@ -0,0 +1,32 @@
+package org.javarosa.core.io;
+
+import java.io.PrintStream;
+
+/**
+ * Used to break dependency directly on System.out and System.err.
+ */
+public class Std {
+
+    public static PrintStream out = System.out;
```

Comment:
This overlaps files I have in pending changes in two branches, but that’s life.

Why not change this to use proper logging now, to avoid having to change all the calls again.